docs/ROADMAP.md

```diff
@@ -91,28 +91,28 @@ New stuff - curl
    of course.
 
 2. Simplify the SMTP command line so that the headers and multi-part content
-   don't have to be constructed before calling curl
+   don't have to be constructed before calling curl.
 
 Improve
 -------
 
-1. build for windows (considered hard by many users)
+1. Build for windows (considered hard by many users).
 
-2. curl -h output (considered overwhelming to users)
+2. Curl -h output (considered overwhelming to users).
```

Comment:
... and certainly not when used in a command line snippet like this.

Reply:
fixed.